.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          # disable issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-pr-message: |
+            This pull request is stale because it has been open for 14 days with no activity.
```

Comment:
So, take this with a grain of salt, because I've only ever written one GH workflow, and I tested it out by mercilessly pushing commits until I got it working :smile: 
It says up top in the `name` section that it checks stale issues and pull requests, but the message here refers only to pull requests, and I assume the below 
```yml
         # disable issues
          days-before-issue-stale: -1
          days-before-issue-close: -1
```
doesn't apply this workflow to issues then, in which case the `name` portions is wrong?
Now, I see you've described all of this in the opening comment of the PR, but does that mean we're going to add support for issues later in this exact workflow, or we'll have a separate one for issues?

Reply:
Yes it is wrong indeed. It is because initially, I configured it for both (issues and PRs), but after some discussion, we decided that we will have it activated only for PRs (see here: https://github.com/ethereum/solidity/pull/13499#issuecomment-1245600327).

As I understood the discussions, for PRs the idea is to consider all currently open PRs (https://github.com/ethereum/solidity/pull/13522#discussion_r970773088), label them as stale, and comment (after 14 days of inactivity), and eventually close them after more 7 days.

For issues, the idea is to exempt issues that already exist and only consider new ones based on their creation/update time. They should not label as stale but commented. They should not be closed but may be moved to an `Icebox` column on the project board.
This could be done using the same GH action that I am using here but adding a new job that is only performed for issues or checking the stale status manually. I started to experiment a bit with the latter here: https://github.com/ethereum/solidity/commit/131d6e53df270f3c8c04e572fd4efef3c8674236